site/components/pages/Pledge/PledgeDashboard/Cards/AssetBalanceCard/index.tsx

```diff
@@ -64,11 +66,14 @@ export const AssetBalanceCard: FC<Props> = (props) => {
   const [balances, setBalances] = useState<Balances | null>(null);
 
   const holdingsByToken = groupBy(props.holdings, "token");
+
+  /** Hide tokens with no balance
+   * @todo hide token with no transaction history*/
   const tokenHoldingAndBalances = map(TOKEN_MAP, (token, key) => ({
     ...token,
     balance: balances && balances[key as BalanceToken],
     holdings: holdingsByToken[token.label],
-  }));
+  })).filter(({ balance }) => !!toNumber(balance));
```

Comment:
built in js `Number` should work fine `!!Number(balance)`

Reply:
Updated